### DISCUSSION TITLE
Random is really random

### DISCUSSION THREAD

**Discussion by Carcinogen72** [quodlibet/quodlibet#4231] [General]
I have the loop button checked, and the random is on, all files, repeat.
The player shows me a song that it is going to play, then plays something completely different.
It's disconcerting, but at least it's random!
I have many mp3 and mp4 files in my library, some are mp4 movie, some mp4's are sound only. The player seems to get more confused when the files are mixed.

Thanks for the great player, 
Hanson
Specs
i5 6 core, @3.6 GHz
Linux Mint 20.2
32GB Ram
GTX 1070
Not sure if this guff will help. But here it is anyways.

Strike the Earth!

**Comment by declension** [comment]
> The player shows me a song that it is going to play, then plays something completely different.

QL doesn't really have an "up next" display so maybe you're looking at something else. The currently selected song in the songlist is more for browsing / editing than controlling playing. The small play icon appears on a row when that song is playing, and if playing in order (no shuffle or repeat) then the next down will be played after, but if random then a random choice from the selected list will be played, depending on which shuffle plugin is activated (the drop down next to shuffle toggle) 

Anywyay if that doesn't solve it, can you record a video?

**Comment by Carcinogen72** [comment]
I'm working on it, haven't forgotten.

Hanson
